packages/genesys-spark-components/src/components/beta/gux-avatar-focusable/gux-avatar-focusable.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, h, JSX, Element, State, Host } from '@stencil/core';
+import { logWarn } from '@utils/error/log-error';
+
+@Component({
+  styleUrl: 'gux-avatar-focusable.scss',
+  tag: 'gux-avatar-focusable-beta',
+  shadow: true
+})
+export class GuxAvatarFocusable {
+  @Element() root: HTMLElement;
+
+  @State()
+  private avatarElement: HTMLGuxAvatarBetaElement;
+
+  componentWillLoad() {
+    this.validateSlot();
+  }
+
+  private validateSlot(): void {
+    const slottedElement = this.root.querySelector('a, button');
+    if (slottedElement) {
+      this.avatarElement = slottedElement.querySelector('gux-avatar-beta');
+      if (!this.avatarElement) {
+        logWarn(this.root, 'Slotted element must contain a gux-avatar');
+      }
+    } else {
+      logWarn(
+        this.root,
+        'An anchor tag or button tag must be slotted into gux-avatar-focusable'
+      );
+    }
+  }
+
+  render(): JSX.Element {
+    return (
+      <Host class={`gux-size-${this.avatarElement?.size ?? 'large'}`}>
```

Comment:
Using `gux-table` as a reference I think this should be
```
<Host gs-avatar-size={this.avatarElement?.size ?? 'large'}>
```

Reply:
Matt explained this to me so no need to respond.